classes/CronJobsForms.php

```diff
@@ -279,21 +279,39 @@ public static function getTasksListValues()
     {
         $id_shop = (int)Context::getContext()->shop->id;
         $id_shop_group = (int)Context::getContext()->shop->id_shop_group;
+        $table_name = _DB_PREFIX_.bqSQL(self::$module->name);
+
+        $select_query = sprintf(
+            "SELECT * FROM `%s` WHERE `id_shop` = '%s' AND `id_shop_group` = '%s'",
```

Comment:
You should use `%d` instead of `%s` for ints, don't you think?
```diff
-`id_shop` = '%s'
+`id_shop` = %d
-`id_shop_group` = '%s'
+`id_shop_group` = %d
```

Reply:
Fixed in https://github.com/PrestaShop/cronjobs/pull/43/commits/17c3dfe202487323a51ff0c177f505358c0334e4